app/src/main/java/org/oppia/app/home/topiclist/PromotedStoryViewModel.kt

```diff
@@ -0,0 +1,43 @@
+package org.oppia.app.home.topiclist
+
+import android.view.View
+import androidx.appcompat.app.AppCompatActivity
+import androidx.databinding.ObservableField
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.ViewModel
+import org.oppia.app.fragment.FragmentScope
+import org.oppia.app.home.HomeActivity
+import org.oppia.app.model.PromotedStory
+import org.oppia.app.viewmodel.ObservableViewModel
+import javax.inject.Inject
+
+// TODO(#283): Add download status information to promoted-story-card.
+
+/** [ViewModel] for displaying a promoted story. */
+@FragmentScope
+class PromotedStoryViewModel @Inject constructor(
+  private val activity: AppCompatActivity
+) : ObservableViewModel() {
+  /**
+   * The retrieved [LiveData] for retrieving topic summaries. This model should ensure only one
+   * [LiveData] is used for all subsequent processed data to ensure the transformed [LiveData]s are
+   * always in sync.
+   */
+
```

Comment:
Nit: remove blank newline here. KDocs should always be directly next to the class member that they are documenting.

Reply:
Done.